src/data/STATUSES/layers/patch7.4.ts

```diff
@@ -7,5 +8,16 @@ export const patch740: Layer<StatusRoot> = {
 		COLLECTIVE_UNCONSCIOUS_MITIGATION: {
 			duration: 10000,
 		},
+
+		BLOODFEST: {
+			id: 5051,
+			name: 'Bloodfest',
+			icon: iconUrl(213623),
+			duration: 8000,
```

Comment:
The new Bloodfest status appears to be a 30s duration for me, mistype or is there a reason this is only set to 8s?

Reply:
it is 30s, mistype indeed.